### PR TITLE
i18n(fr): Update `components.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `components.mdx` based on these commits faa70de584bf596fdd7184c4a8622d67d1410ecf & dc42569bddfae2c48ea60c0dd5cc70643a129a68

### CODE REVIEW

File: docs/src/content/docs/fr/guides/components.mdx [hunk 1]
```diff
@@ -33,6 +33,20 @@ import AnotherComponent from '../../components/AnotherComponent.astro';
...
+
+Starlight applique des styles par défaut à votre contenu Markdown, par exemple en ajoutant une marge entre les éléments.
+Si ces styles entrent en conflit avec l'apparence de votre composant, définissez la classe `not-content` sur votre composant pour les désactiver.
+
+```astro
+---
+// src/components/Example.astro
+---
+<div class="not-content">
+  <p>Not impacted by Starlight’s default content styling.</p>
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:46]:
```suggestion
  <p>Contenu non affecté par les styles par défaut de Starlight.</p>
```

File: docs/src/content/docs/fr/guides/components.mdx [hunk 2]
```diff
@@ -109,3 +123,28 @@ Ajoutez l'attribut `stagger` pour décaler verticalement la deuxième colonne de
 ```
 
 :::
+
+### Icon
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:127]: In the current translation, `Tabs` is translated to `Onglets`, `Cards` is not translated. It looks like most translations translate the component names so I guess we should do the same. If so, that would mean translating `Cards` too I guess. Any thoughts?

```suggestion
### Icônes
```

File: docs/src/content/docs/fr/guides/components.mdx [hunk 3]
```diff
@@ -68,7 +82,7 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 Vous pouvez afficher du contenu dans une boîte correspondant aux styles de Starlight en utilisant le composant `<Card>`.
 Enveloppez plusieurs cartes dans le composant `<CardGrid>` pour afficher les cartes côte à côte lorsqu'il y a suffisamment d'espace.
 
-Une `<Card>` nécessite un `titre` et peut éventuellement inclure un attribut `icon` fixé au nom de [l'une des icônes intégrées de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Icons.ts).
+Une `<Card>` nécessite un `title` et peut optionellement inclure un attribut `icon` fixé au nom de [l'une des icônes intégrées de Starlight](#all-icons).
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:85]:
```suggestion
Une `<Card>` nécessite un `title` et peut optionnellement inclure un attribut `icon` fixé au nom de [l'une des icônes intégrées de Starlight](#toutes-les-icônes).
```

File: docs/src/content/docs/fr/guides/components.mdx [hunk 4]
```diff
@@ -109,3 +123,28 @@ Ajoutez l'attribut `stagger` pour décaler verticalement la deuxième colonne de
...
 :::
+
+### Icon
+
+import { Icon } from '@astrojs/starlight/components';
+import IconsList from '../../../components/icons-list.astro';
+
+Starlight fournit un ensemble d'icônes courantes que vous pouvez afficher dans votre contenu à l'aide du composant `<Icon>`.
+
+Chaque `<Icon>` nécessite un [`name`](#all-icons) et peut optionellement inclure un attribut `label`, `size` et `color`.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/components.mdx:134]:
```suggestion
Chaque `<Icon>` nécessite un [`name`](#toutes-les-icônes) et peut optionnellement inclure un attribut `label`, `size` et `color`.
```

**Review by HiDeoo — COMMENTED** [review]
Left a few suggestions for invalid anchors and some minor changes, if you have any thoughts on those.

**Comment by dreyfus92** [comment]
Merci pour le code review @HiDeoo 😁

**Review by delucis — APPROVED** [review]
Looks good to me! Thank you @dreyfus92 — our multilingual contributor 🥳 — and thanks for reviewing @HiDeoo.

**Pull request merged**